sw/airborne/modules/computer_vision/video_thread.c

```diff
@@ -80,6 +80,9 @@ static void *video_thread_function(void *data)
 {
   struct video_config_t *vid = (struct video_config_t *)data;
 
+  char print_tag[80];
+  sprintf(print_tag, "video_thread-%s", vid->dev_name);
```

Comment:
Copying the reference would be fine, but really copying the image costs time. Can the img ref be used safely at the moment even when the processing will end after a new image comes in? Or is the image memory block freed before?